website/blog/2023-01-04-ferretdb-beta-version-authentication.md

```diff
@@ -0,0 +1,118 @@
+```
+
+We've also added support for `ordered` inserts, which allows you to insert data in the exact order it comes in.
+
+```js
+db.collection.insert( { "field1": "value1" }, { ordered: true } )
+```
+
+## Bug fixes and enhancements
+
+This release includes a bug fix that addresses an issue with the `$inc` operator when used with `unset` documents.
+We found that, in certain cases, an invalid value of the `$inc` operator was updating `unset` documents with non-numeric values when it should have instead returned an error.
+This release should prevent unexpected updates on `unset` documents, and likewise, improve the overall stability and reliability of FerretDB.
+
+The release includes one enhancement, which is updating our building documentation.
+We have documented our build process and Go build tags for those who use embedded packages or build FerretDB themselves.
+Please [check here](https://github.com/FerretDB/FerretDB/blob/main/README.md#building-and-packaging) for further details.
+
+## Documentation
+
+Our documentation is not left out on the list of improvements!
+Specifically, we've updated it to include descriptions and examples for element query operators, array query operators, and comparison and logical query operators.
+
+These changes help make our documentation more comprehensive and user-friendly, which should hugely benefit our community and users alike.
+
+## Other changes
+
+In this latest release, we've made some changes to our supported features, which include discontinuing support for `$elemMatch` and `$slice` projection operators due to existing technical issues.
+However, this change is not a cause for concern, merely a signal that we will be focusing on other features and improvements that are of greater priority in [our roadmap](https://github.com/orgs/FerretDB/projects/2).
+This change will not affect the support for the `$elemMatch` query operator.
+For more information on the supported operators and commands, please check out our [updated documentation](https://docs.ferretdb.io/reference/supported_commands/).
+
+Please find more details about the new features and changes in the beta version [here](https://github.com/FerretDB/FerretDB/releases/tag/v0.8.0).
+
+## And to our amazing community and users
+
+Once again, we want to take a moment to express our gratitude to everyone who has been part of the FerretDB journey so far - either as a user, contributor, supporter, or admirer.
+Not to mention the growing number of partnerships, compatible applications, and all the contributions from our incredible community.
+In 2022, we had:
+
+* 👨🏻‍💻 Over 40 code contributors with more than 130 merged pull requests from our community of contributors
+* ⭐️ 5.1k Stars on GitHub(Have you given us a Star yet? Be sure to [Star us on GitHub](https://github.com/FerretDB/FerretDB).)
+* ⏫ More than 100 Docker image downloads
+* 🎤 And, we actively participated in several events, including Percona Live,
```

Comment:
Stats are great, but event participation has nothing to do with the release, I would leave it out